opta/commands/secret.py

```diff
@@ -178,7 +179,45 @@ def update(
     update_manual_secrets(layer.name, {secret: str(value)})
     __restart_deployments(no_restart, layer.name)
 
-    print("Success")
+    logger.info("Success")
+
+
+@secret.command()
+@click.argument("secret")
+@restart_option
+@config_option
+@env_option
+@input_variable_option
+@local_option
+def remove(
+    secret: str,
+    env: Optional[str],
+    config: str,
+    no_restart: bool,
+    local: Optional[bool],
+    var: Dict[str, str],
+) -> None:
+    """Update a given secret of a k8s service by removing a manual entry
```

Comment:
Let's stay away from using _update_ since we already have `opta secrets update`
-> `Delete a secret key from a k8s service`